src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    public enum Operation {
+        B,
+        P,
+        R,
+        S
+    }
```

Comment:
The enum Operation uses single-letter names (B, P, R, S) which are not descriptive. Consider using full names for operations to make the code more readable. For example, BUY, PURCHASE, RETURN, SUPPLY could be used instead.